src/controllers/users/users.routes.js

```diff
@@ -0,0 +1,131 @@
+const router = require('express').Router();
+const bcrypt = require('bcryptjs');
+const db = require('./users.models.js');
+const jwt = require('../../utils/jwt.js');
+
+router.post('/login',
+  loginInputs,
+  loginDataType,
+  usernameLowerCase,
+  async (req, res, next) => {
+    console.log(req.body);
+    const { username, password } = req.body;
+    try {
+      const user = await db.validate(username)
+      if (user && bcrypt.compareSync(password, user.password)) {
+        const token = await jwt.sign({ id: user.id, username: user.username })
```

Comment:
Great job w/ the async/await syntax (as opposed to the `.then().catch()`. Just reads a lot cleaner and is the 'newer' syntax.